code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -183,41 +183,41 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	return ..()
 
 /datum/game_mode/dynamic/send_intercept()
-	. = "<b><i>Central Command Status Summary</i></b><hr>"
+	. = "<b><i>Podsumowanie Stanu Bezpieczeństwa Stacji</i></b><hr>"
 	switch(round(threat_level))
 		if(0 to 19)
 			update_playercounts()
 			if(!current_players[CURRENT_LIVING_ANTAGS].len)
-				. += "<b>Peaceful Waypoint</b></center><BR>"
-				. += "Your station orbits deep within controlled, core-sector systems and serves as a waypoint for routine traffic through Nanotrasen's trade empire. Due to the combination of high security, interstellar traffic, and low strategic value, it makes any direct threat of violence unlikely. Your primary enemies will be incompetence and bored crewmen: try to organize team-building events to keep staffers interested and productive."
+				. += "<b>Bezpieczny Punkt</b></center><BR>"
+				. += "Wasza stacja znajduje się w bardzo dobrze chronionym regionie centralnym. Stacja jest często odwiedzana przez patrole Nanotrasenu. Wasz sektor kipi życiem oraz handlem.  Z powodu kombinacji wysokiego poziomu bezpieczeństwa, bezpośredniego szlaku handlowego , częstych podróży międzygwiezdnych oraz małej wartości strategicznej, bezpośrednie zagrożenia są bardzo bardzo mało prawdopodobne. Największym przeciwnikiem będzie niekompetencja oraz znudzona załoga: spróbujcie zorganizować cel stacji oraz imprezy w barze trzymać produktywność oraz zadowolenie w wysokiej formie."
```

Comment:
- Punkt => Przyczółek, lepiej pasuje w tym kontekście
- "znajduje się w bardzo dobrze chronionym regionie centralnym. Stacja jest często odwiedzana przez patrole Nanotrasenu." => "orbituje głęboko pomiędzy systemami głównych, kontrolowanych sektorów i jest węzłem w rutynowym ruchu przez imperium handlowe Nanotrasenu."
- "Wasz sektor kipi życiem oraz handlem." - to wyrzucić